<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1356</width>
    <height>810</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>高分遥感图像处理系统</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="anomalyDetectionTab">
       <attribute name="title">
        <string>异常检测</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="4,6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
          <item>
           <widget class="QGroupBox" name="groupBox_6">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>原图</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QGraphicsView" name="ADOriginImageContainer"/>
             </item>
             <item>
              <widget class="QPushButton" name="ADChoseImageBtn">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>选择图片</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="ADBtn">
              <property name="text">
               <string>异常检测</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ADResetBtn">
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>异常检测结果</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_13">
             <item>
              <widget class="QGraphicsView" name="ADResultContainer"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="segmentationTab">
       <attribute name="title">
        <string>语义分割</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="4,6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0">
          <item>
           <widget class="QGroupBox" name="groupBox_4">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>原图</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QGraphicsView" name="SSOriginImageContainer"/>
             </item>
             <item>
              <widget class="QPushButton" name="SSChoseImageBtn">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                </font>
               </property>
               <property name="text">
                <string>选择图片</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="SSBtn">
              <property name="text">
               <string>语义分割</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="SSResetBtn">
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_14">
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>语义分割结果</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QGraphicsView" name="SSResultContainer"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="changeDetectionTab">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <attribute name="title">
        <string>变化检测</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,0,0">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>原图1</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QGraphicsView" name="CDOriginImageContainer1"/>
             </item>
             <item>
              <widget class="QPushButton" name="CDChoseImageBtn1">
               <property name="text">
                <string>选择图片</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>原图2</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QGraphicsView" name="CDOriginImageContainer2"/>
             </item>
             <item>
              <widget class="QPushButton" name="CDChoseImageBtn2">
               <property name="text">
                <string>选择图片</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="CDBtn">
              <property name="text">
               <string>变化检测</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="CDResetBtn">
              <property name="text">
               <string>重置</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="title">
             <string>变化检测结果</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QGraphicsView" name="CDResultContainer">
               <property name="font">
                <font>
                 <pointsize>16</pointsize>
                </font>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="removalTab">
       <attribute name="title">
        <string>去云雾</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>11</x>
          <y>8</y>
          <width>531</width>
          <height>731</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,0">
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="title">
            <string>原图</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QGraphicsView" name="removalOriginImageContainer"/>
            </item>
            <item>
             <widget class="QPushButton" name="removalChoseImageBtn">
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>选择图片</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QPushButton" name="removalBtn">
             <property name="text">
              <string>去云雾</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removalResetBtn">
             <property name="text">
              <string>重置</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>541</x>
          <y>13</y>
          <width>781</width>
          <height>731</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <item>
          <widget class="QGroupBox" name="groupBox_9">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="title">
            <string>去云雾结果</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <widget class="QGraphicsView" name="removalResultContainer"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1356</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CDBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>changeDetection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>144</x>
     <y>737</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CDChoseImageBtn2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>665</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CDResetBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetCDContainer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>739</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CDChoseImageBtn1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>268</x>
     <y>353</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SSChoseImageBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>684</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ADChoseImageBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>354</y>
    </hint>
    <hint type="destinationlabel">
     <x>217</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SSBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>anomalyDetection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>748</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SSResetBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetADContainer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>737</y>
    </hint>
    <hint type="destinationlabel">
     <x>420</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ADBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>imageFusion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>743</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>-35</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ADResetBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetIFContainer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>746</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>-151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeDetection()</slot>
  <slot>loadImage()</slot>
  <slot>resetCDContainer()</slot>
  <slot>anomalyDetection()</slot>
  <slot>resetADContainer()</slot>
  <slot>resetIFContainer()</slot>
  <slot>imageFusion()</slot>
 </slots>
</ui>
